_posts/en/newsletters/2023-04-26-newsletter.md

```diff
@@ -0,0 +1,147 @@
+    #60][news60 bloom]); this effectively makes BIP35 `mempool` also
+    disabled by default.
+
+    Clark's Bitcoin Core PR received support from within the project,
+    although some supporters think BIP37 bloom filters should be removed
+    first.  On the mailing list, the only [reply][harding mempool] as of
+    this writing noted that lightweight clients that connect to their
+    own trusted node can currently use BIP35 and BIP37 to learn about
+    unconfirmed transactions in a much more bandwidth efficient manner
+    than any other method that's currently easily available through
+    Bitcoin Core.  The respondent suggested that Bitcoin Core provide an
+    alternative mechanism before removing the current interface.
+
+    Additional feedback is requested from anyone using the BIP35
+    `mempool` message for any purpose.  You can reply to either the
+    mailing list post or the PR linked previously.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [How many sigops are in the invalid block 783426?]({{bse}}117837)
+  Vojtěch Strnad provided a script which iterates through all transactions in a
+  block and counts the [sigops]({{bse}}117359) and notes that there were 80,003 sigops in the
+  block, making it [invalid][max sigops].
+
+- [How would an adversary increase the required fee to replace a transaction by up to 500 times?]({{bse}}117734)
+  In referencing a draft BIP for [ephemeral anchors][topic ephemeral anchors],
+  Michael Folkson asks how the referenced 500 times increase in required fees
+  for a transaction replacement could occur. Antoine Poinsot gives an example
+  of how an attacker could use the [Replace-By-Fee (RBF)][topic rbf] fee bumping
+  rules to require additional replacement transactions to pay significantly
+  higher fees.
+
+- [Best Practices with Multiple CPFPs & CPFP + RBF?]({{bse}}117877)
+  Sdaftuar explains considerations around using RBF and [Child Pays For Parent
+  (CPFP)][topic cpfp] fee bumping techniques in the scenario that an initial
+  CPFP fee bumping attempt failed to offer a sufficient feerate to get the
+  initial transaction confirmed.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Core Lightning 23.05rc1][] is a release candidate for the next
+  version of this LN node.
+
+- Bitcoin Core 25.0RC1 FIXME:harding
```

Comment:
Note, dropping the Bitcoin Core RC.  As of this writing, it hasn't been uploaded to bitcoincore.org and I don't think it's urgent that we announce it this week.